packages/xo-server-sdn-controller/src/index.js

```diff
@@ -111,6 +111,20 @@ function createPassword() {
   return Array.from({ length: 16 }, _ => sample(chars)).join('')
 }
 
+// -----------------------------------------------------------------------------
+
+// Used in openssl cert creation because the error field of the callbabck is set to `false` if no error happened.
+const fromCallbackFalsyError = (fn, ...args) =>
+  new Promise((resolve, reject) => {
+    fn(...args, (error, result) => {
+      if (error) {
+        reject(error)
+      } else {
+        resolve(result)
+      }
+    })
+  })
```

Comment:
Maybe I should update `promise-toolbox/fromCallback` to handle this case, this pattern is probably also used in other libs.

Reply:
I was thinking about that.
An argument could be the the test to choose how to decide if there's an error or not with by default
`error != undefined`?

Reply:
I don't think that's necessary, this case is not the strange.

Reply:
Implemented in `promise-toolbox@0.14`.